pallets/settlement/src/lib.rs

```diff
@@ -971,6 +1071,97 @@ impl<T: Trait> Module<T> {
         Ok(instruction_counter)
     }
 
+    pub fn base_confidential_add_instruction(
+        did: IdentityId,
+        venue_id: u64,
+        settlement_type: SettlementType<T::BlockNumber>,
+        valid_from: Option<T::Moment>,
+        legs: Vec<Leg<T::Balance>>,
+    ) -> Result<u64, DispatchError> {
+        // Check whether the no. of legs within the limit or not.
+        ensure!(
+            u32::try_from(legs.len()).unwrap_or_default() <= T::MaxLegsInAInstruction::get(),
```

Comment:
CC core team: @maxsam4 @satyamakgec @vkomenda regarding changing `MaxLegsInAInstruction` to `MaxLegsInAnInstruction`  (A is changed to An)

Reply:
This is just a spelling fix (read it out loud as a sentence); which should be trivial to fix, since its an associated type, and doesn't need migration or anything like that.